src/content/peerconnection/pc1/js/main.js

```diff
@@ -68,6 +71,7 @@ function gotStream(stream) {
 
 function start() {
   trace('Requesting local stream');
+  webAudio.start();
```

Comment:
I had an empty start method as part of webaudioextended.js at one point, but removed it. I can add it back if needed.